packages/slides-frontend/src/utils/helperFunctions.js

```diff
@@ -0,0 +1,33 @@
+
+const canDeleteImageFromBackend = (imageName, slidesStringified) => {
+    const slides = JSON.parse(slidesStringified);
+    let counter = 0;
+    slides.map(slide =>
+        slide.itemsArray.map(item => {
+            if (item.type === "IMAGE" && item.Src === imageName) {
+                counter = counter + 1;
+            }
+        })
+    );
+    return counter === 1;
+}
+
+const deepCopyFunction = inObject => {
+    let outObject, value, key;
+    if (typeof inObject !== "object" || inObject === null) {
+      return inObject; // Return the value if inObject is not an object
```

Comment:
I think some of these comments are not really necessary. The code is pretty self-explanatory.